debian/control

```diff
@@ -2,8 +2,8 @@ Source: anbox
 Section: misc
 Priority: optional
 Maintainer: Simon Fels <morphis@gravedo.de>
-Build-Depends: debhelper (>= 7), dkms, dh-systemd
-Standards-Version: 3.9.7
```

Comment:
Why is this bumped?  We're still supporting older platforms which may not have the latest version of the standard or debhelper

Reply:
Hm, I was thinking debhelper was at that version for xenial. However thanks for pointing this out. Leaving it at "9" is right as we still want to support xenial.